src/mnemonics/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ add_library(mnemonics
 
 target_link_libraries(mnemonics
   PRIVATE
+    sodium
```

Comment:
Why is this here? Nothing in `mnemonics` should touch anything to do with sodium.

Reply:
I'd rather fix the include problem.  I think it should be fine to remove the include (and, in fact, *all* the includes) from `crypto/generic-ops.h`.  This header only has macros in it, which don't need a header at all: instead push the header requirement to whatever is using the macros (which most likely already have the needed headers).